applications/deployments/deployment-strategies.adoc

```diff
@@ -6,24 +6,14 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-A _deployment strategy_ is a way to change or upgrade an application. The aim is to make the change without downtime in a way that the user barely notices the improvements.
+_Deployment strategies_ are used to change or upgrade applications without downtime, in a way that users barely notice a change.
```

Comment:
```suggestion
_Deployment strategies_ are used to change or upgrade applications without downtime so users barely notice a change.
```
- Maybe this edit would make the sentence a bit more streamlined? Your call whether to make the change.